### ISSUE TITLE
does the "centre feature vectors" achieved by the average of n-512 vectors?

### ISSUE THREAD

**Issue by ReverseSystem001** [deepinsight/insightface#1033]
if one person has n face images. After they through the net, we can achieve n-512 vectors, but we should choose one vector to represent a person(have many face images). so we use the **center feature vector** represent the person. is right? But how do you achieve the **center feature vector**.
is it achieved by the average of n-512 vectors???
hope your replay!

**Comment by Walstruzz** [comment]
In arcface, **center feature vector** is a learnable matrix here.
https://github.com/deepinsight/insightface/blob/fdedc4a396987ab785ca4ef61908c2eb643b7163/recognition/train.py#L78-L79

**Comment by ReverseSystem001** [comment]
@Walstruzz  the 3.1 of the paper said that: **Note that, overlap identities between the training set and the test set are remove for strict evaluations, and we only use a single crop for all testing**. now I want to ensure there is no training set overlaps with FaceScrub identities. so I need to remove identities from my training set whose cosine similarity is larger than 0.4 with any FaceScrub identity by comparing their **centre feature vectors**. I mean this centre feature vectors. are you sure it is a learnable matrix here like you said???
